model/src/w3sic5md.F90

```diff
@@ -573,6 +633,22 @@ SUBROUTINE W3IC5WNCG(WN_R, WN_I, CG, HICE, IVISC, RHOI, ISMODG, &
   END SUBROUTINE W3IC5WNCG
   !/ ------------------------------------------------------------------- /
   !/
+  !>
+  !> @brief Calculate the complex wavenumber for waves in ice according to
+  !>  three different sea ice models.
```

Comment:
any reason that:   " i.e., FS, RP and M2 (see Liu et al. 2020)" is left out here?

Reply:
For this there is two reasons why that part got cut.  The first is it's getting to be a bit long for the brief sentence, in my opinion.  I could see that being debatable, though. Next it has a couple periods '.' in it, and the first period encountered in the brief terminates the sentence.  So in this case the sentence would be truncated at 'i.' in the 'i.e., '.  

You are supposed to be able to escape the period (with a '\'), but experimenting with it I did not have luck with that, so I have been re-wording in some cases to avoid non-terminal periods in the brief sentence.  This is a case where if I followed my rough guidelines I should have included that end part in the details.  I'll make that fix to be consistent.